src/containers/TopicArticlePage/components/StructureFunctionButtons.jsx

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
Denne komponenten ser jo veldig lik ut som StructureButtons lenger opp?

Reply:
Ja den er lik, men ene er topic-article, så er ulike knapper. Tenkte det var best å ha to ulike